pkg/apis/pipeline/v1/taskref_validation.go

```diff
@@ -55,14 +54,3 @@ func (ref *TaskRef) Validate(ctx context.Context) (errs *apis.FieldError) {
 	}
 	return
 }
-
-func validateResolutionParamTypes(params []Param) (errs *apis.FieldError) {
-	for i, p := range params {
-		if p.Value.Type == ParamTypeArray || p.Value.Type == ParamTypeObject {
-			errs = errs.Also(apis.ErrGeneric(fmt.Sprintf("remote resolution parameter type must be %s, not %s",
-				string(ParamTypeString), string(p.Value.Type))).ViaIndex(i))
-		}
-	}
-
-	return errs
-}
```

Comment:
I think this was added in https://github.com/tektoncd/pipeline/pull/5387 and it appears to be purposefully different in v1 than in v1beta1-- @abayer can you comment here?

Reply:
This is added at pipelinerunref_validation as it is duplicated. This might be an earlier issue while it was in taskrunref when v1 CRDs are created.

Reply:
Yeah, when we added the resolution code, `v1/pipelineref_validation.go` didn't exist yet, and once it did, I didn't bother moving the code around.